### PR TITLE
feat: add init command

### PR DESCRIPTION
**Pull request by mshaaban0**
## Summary

Add `init` command basic structure

## How to verify

* Clone the branch locally
* run `npm link && npm tsc:watch`
* then run `contentful init`
* You should see Contentful welcome message

## Screenshots:
Screenshot 2022-08-02 at 03 07 51

### CODE REVIEW

**Comment by marcolink** [comment]
Hi @mshaaban0,

out of curiosity, what is the command doing? 
And how will it be different from the old `guide` command?

**Comment by mshaaban0** [comment]
Hey @marcolink 
This will replace the guide command. The main goal for init is to onboard users until they make their first CDA request, show different options with examples to consume content 'ContentfulJS, GraphQL, REST API', and potentially plug in [starters](https://github.com/contentful/starters) later on.

`guide` was only creating a new space with predefined content and Gatsby site, which actually breaks because of undefined variables (spaceId, access token). 

`init` will provide different options and choice to add predefined content or start from scratch, with links to docs and GraphiQL app.

**Comment by marcolink** [comment]
@mshaaban0 thanks for the explanation - looking forward to see it in action 😎

File: lib/config.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-    'login',
-    'snake',
     'config add',
     'config list',
     'config remove',
+    'init',
+    'logout',
+    'login',
+    'organization list',
+    'snake',
```
Comment (bgutsol) [inline lib/config.js:18]: now it's gonna be hidden under auth?

**Review by bgutsol — APPROVED** [review]
LGTM 👍

**Pull request merged**